sdk/formrecognizer/ai-form-recognizer/src/models.ts

```diff
@@ -733,7 +763,29 @@ export interface FormTrainResult {
   /**
    * Errors returned during training operation.
    */
-  errors?: ErrorInformation[];
+  errors?: FormRecognizerError[];
+}
+
+/**
+ * Basic custom model information.
+ */
+export interface CustomFormModelInfo {
+  /**
+   * Model identifier.
+   */
+  modelId: string;
+  /**
+   * Status of the model.
+   */
+  status: ModelStatus;
```

Comment:
Other languages are calling the type `CustomFormModelStatus` instead of `ModelStatus

Reply:
Good catch!  Updated.  Thanks for reviewing!